front/src/modules/trainschedule/components/ManageTrainSchedule/TrainAddingSettings.tsx

```diff
@@ -19,6 +22,8 @@ export default function TrainAddingSettings() {
   const debouncedTrainStep = useDebounce(trainStep, 500);
   const debouncedTrainCount = useDebounce(trainCount, 500);
   const debouncedTrainDelta = useDebounce(trainDelta, 500);
+  const { updateTrainStep, updateTrainCount, updateTrainDelta } =
+    useOsrdActions() as operationalStudiesConfSliceActionsType;
```

Comment:
Could you group it with the other custom hook line 15 :)

Reply:
Done